src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.Windows.cs

```diff
@@ -238,23 +230,20 @@ private static void StartHelper(NetworkAddressChangedEventHandler caller, bool c
 
                     if (!s_isPending)
                     {
-                        int length;
-                        SocketError errorCode;
-
                         if (Socket.OSSupportsIPv4 && (startIPOptions & StartIPOptions.StartIPv4) != 0)
                         {
-                            s_registeredWait = ThreadPool.RegisterWaitForSingleObject(
+                            ThreadPool.RegisterWaitForSingleObject(
```

Comment:
Can you remind me why we don't care about saving this field?

Reply:
We never do anything with it :)  It's only benefit would be unregistering, and we're not doing that.  Further, we're actually overwriting the IPv4 one with the IPv6, so obviously it's not needed ;)